internal/mode/static/nginx/conf/nginx-plus.conf

```diff
@@ -26,6 +26,21 @@ http {
   sendfile on;
   tcp_nopush on;
 
+  server {
+    listen 8765;
```

Comment:
we can bind to 127.0.0.1:8765 too

Reply:
I'm guessing we can then omit `allow` and `deny`?

Reply:
Actually, we would still need those, right? Because that refers to where the client traffic comes from.

Reply:
yep